IPython/core/tests/test_formatters.py

```diff
@@ -347,8 +351,8 @@ def __getattr__(self, key):
     call_hat = CallableMagicHat()
     with capture_output() as captured:
         result = f(call_hat)
-    
-    nt.assert_equal(result, None)
+
+    assert result == None
```

Comment:
Nit, not sure if valid for tests / the style: flake8 complains with `E711: comparison to None should be 'if cond is None:' ` on such lines.

```suggestion
    assert result is None
```